src/sentry/integrations/msteams/notify_action.py

```diff
@@ -68,6 +70,30 @@ def __init__(self, *args, **kwargs):
     def is_enabled(self):
         return self.get_integrations().exists()
 
+    def after(self, event, state):
+        integration_id = self.get_option("team")
+        channel = self.get_option("channel_id")
+
+        try:
+            pass
```

Comment:
@KKaempen I think this `pass` should be removed, yea?

Reply:
Yeah, I'm not sure how I left that in there.